website/docs/r/managed_devops_pool.html.markdown

```diff
@@ -0,0 +1,292 @@
+}
+
+resource "azurerm_managed_devops_pool" "example" {
+  name                           = "example"
+  resource_group_name            = azurerm_resource_group.example.name
+  location                       = azurerm_resource_group.example.location
+  dev_center_project_resource_id = azurerm_dev_center_project.example.id
+  maximum_concurrency            = 1
+
+  organization_profile {
+    kind = "AzureDevOps"
+
+    organizations {
+      parallelism = 1
+      url         = "https://dev.azure.com/example"
+    }
+  }
+
+  agent_profile {
+    kind = "Stateless"
+
+    resource_predictions_profile {
+      kind                  = "Automatic"
+      prediction_preference = "Balanced"
+    }
+  }
+
+  fabric_profile {
+    kind = "Vmss"
+
+    sku {
+      name = "Standard_D2ads_v5"
+    }
+
+    images {
+      resource_id = "/Subscriptions/00000000-0000-0000-0000-000000000000/Providers/Microsoft.Compute/Locations/australiaeast/publishers/canonical/artifacttypes/vmimage/offers/0001-com-ubuntu-server-focal/skus/20_04-lts-gen2/versions/latest"
+      buffer      = "*"
+    }
+
+    storage_profile {
+      os_disk_storage_account_type = "Standard"
+
+      data_disks {
+        disk_size_gb = 1
+        drive_letter = "B"
+      }
+    }
+
+    network_profile {
+      subnet_id = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/group1/providers/Microsoft.Network/virtualNetworks/vnet1/subnets/subnet1"
+    }
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `agent_profile` - (Required) An `agent_profile` block as defined below.
+
+* `dev_center_project_resource_id` - (Required) The ID of the Dev Center project.
+
+* `fabric_profile` - (Required) A `fabric_profile` block as defined below.
+
+* `location` - (Required) The Azure Region where the Managed DevOps Pool should exist. Changing this forces a new Managed DevOps Pool to be created.
+
+* `maximum_concurrency` - (Required) Defines how many resources can there be created at any given time.
+
+* `name` - (Required) The name which should be used for this Managed DevOps Pool. Changing this forces a new Managed DevOps Pool to be created.
+
+* `organization_profile` - (Required) An `organization_profile` block as defined below.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Managed DevOps Pool should exist. Changing this forces a new Managed DevOps Pool to be created.
+
+---
+
+* `identity` - (Optional) An `identity` block as defined below.
+
+* `tags` - (Optional) A mapping of tags which should be assigned to the Managed DevOps Pool.
+
+---
+
+An `agent_profile` block supports the following:
+
+* `kind` - (Required) Defines the type of agent profile. Accepted values are: `Stateful` and `Stateless`.
+
+* `grace_period_time_span` - (Optional) How long should the machine be kept around after it ran a workload when there are no stand-by agents. The maximum is one week. This is needed when kind is `Stateful`.
```

Comment:
what format is this in? seconds? iso duration?
```suggestion
* `grace_period_in_seconds` - (Optional) How long should the machine be kept around after it ran a workload when there are no stand-by agents. The maximum is one week. This is needed when kind is `Stateful`.
```
or 
```suggestion
* `grace_period_Duration` - (Optional) How long should the machine be kept around after it ran a workload when there are no stand-by agents in ISO8601 duration format, for example `P14D`. The maximum is one week. This is required when kind is `Stateful`.
```

Reply:
This is our public docs for our reference and want to keep it in the same way.
However, I have updated the description. please check.

https://learn.microsoft.com/en-us/azure/devops/managed-devops-pools/configure-scaling?view=azure-devops&tabs=azure-portal
![image](https://github.com/user-attachments/assets/6822928b-e899-49a2-8544-a936f4f9ba0c)